PollBuddy-Server/frontend/src/pages/RegisterWithPollBuddy/RegisterWithPollBuddy.js

```diff
@@ -76,8 +76,10 @@ class RegisterWithPollBuddy extends Component {
             To create an account, fill in the text boxes, then press submit.
           </p>
           <MDBContainer className="form-group">
-            <label htmlFor="nameText">Name:</label>
-            <input placeholder="SIS Man" className="form-control textBox" id="nameText" 
+            <label htmlFor="firstnameText">Name:</label>
+            <label htmlFor="lastnameText">Name:</label>
+            <input placeholder="SIS" className="form-control textBox" id="firstnameText"/>
+            <input placeholder="Man" className="form-control textBox" id="lastnameText"/>
               onChange={(evt) => { this.setState({username: evt.target.value}); }}/>
```

Comment:
This onChange part is actually part of the tag. You'll want to include it for each I believe as that's how the input validation hooks in. Duplicate the onChange part for each, and adjust the relevant fields.